src/test/java/seedu/academyassist/testutil/TypicalPersons.java

```diff
@@ -32,8 +32,8 @@ public class TypicalPersons {
             .withTags("friends").build();
     public static final Person BENSON = new PersonBuilder().withName("Benson Meier")
             .withAddress("311, Clementi Ave 2, #02-25").withSubjects("Maths").withEmail("johnd@example.com")
```

Comment:
"Maths" might be filtered out by `SampleDataUtil::getSubjectSet`. Perhaps consider changing it to "Math".

Reply:
Good catch, thanks for pointing this out!